lib/dpl/provider/ops_works.rb

```diff
@@ -40,6 +40,7 @@ def check_auth
       end
 
       def custom_json
+        options[:custom_json] = YAML.load(options[:custom_json]) if options[:custom_json].is_a? String
```

Comment:
Are you loading option value named `custom_json` with the YAML loader? That seems incorrect to me.